### ISSUE TITLE
frontend breadcrumb: double entry and a11y

### ISSUE THREAD

**Issue by lizardat** [contao/contao#9588]
### Affected version(s)

5.7.1

### Description

The breadcrumb module produces the current page twice in the list:
e.g. First level > second level > Page > Page

also, the last entry is marked in mod_breadcrumb as <li .... aria-current="page">...</li>. This current page should be with a link as all other entries, as aria features are supposed to be used on interactive elements only.

See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-current
